scarb/src/ops/publish.rs

```diff
@@ -10,6 +10,7 @@ use crate::sources::RegistrySource;
 pub struct PublishOpts {
     pub index_url: Url,
     pub allow_dirty: bool,
+    pub verify: bool,
```

Comment:
hmm, a nitpick: what about embedding `PackageOpts` here instead of copy-pasting its fields?

we could also share `clap` arguments that way, thanks to `#[clap(flatten)]`